src/main/java/controller/PlayerController.java

```diff
@@ -0,0 +1,206 @@
+        Player sGA = new Player("Shai Gilgeous-Alexander", 2, "Canada", teamThunder, 11, 6);
+        Player bigScience = new Player("Jonas Valančiūnas", 17, "Lithuania", teamWizards, 5, 13);
+        Player deAndre = new Player("DeAndre Ayton", 2, "Bahamas", teamTrailBlazers, 1, 6);
+        Player lukaMagic = new Player("Luka Dončić", 77, "Slovenia", teamMavericks, 3, 6);
+        Player rjBarrett = new Player("RJ Barrett", 9, "Canada", teamRaptors, 3, 5);
+        Player domantas = new Player("Domantas Sabonis", 11, "Lithuania", teamKings, 11, 8);
+        Player joker = new Player("Nikola Jokić", 15, "Serbia", teamNuggets, 41, 10);
+        Player dBook = new Player("Devin Booker", 1, "USA", teamSuns, 13, 9);
+        Player lauriBird = new Player("Lauri Markkanen", 23, "Finland", teamJazz, 7, 7);
+        Player banchero = new Player("Paolo Banchero", 5, "Italy", teamMagic, 1, 2);
+        Player schroder = new Player("Dennis Schröder", 17, "Germany", teamNets, 17, 11);
+        Player haliburton = new Player("Tyrese Haliburton", 0, "USA", teamPacers, 12, 4);
+        Player laMelo = new Player("LaMelo Ball", 1, "USA", teamHornets, 3, 4);
+        Player theSystem = new Player("James Harden", 1, "USA", teamClippers, 3, 15);
+        Player antMan = new Player("Anthony Edwards", 5, "USA", teamTimberwolves, 1, 4);
+        Player jaG12 = new Player("Ja Morant", 12, "USA", teamGrizzlies, 2, 5);
+        Player ingram = new Player("Brandon Ingram", 14, "USA", teamPelicans, 2, 8);
+
+        Collections.addAll(players,
+                tacoJay, leBron, theChef, flight8, wemby, joJo, motorCade, jimBucket,
+                jayBrunson, alpySen, greekFreak, spida, swissBank, sGA, bigScience, deAndre,
+                lukaMagic, rjBarrett, domantas, joker, dBook, lauriBird, banchero, schroder,
+                haliburton, laMelo, theSystem, antMan, jaG12, ingram
+        );
+    }
+
+    /**
+     * This method normalizes the user's input in case the String 'name' entered
+     * does not exactly match the diacritics contained in the attribute of the 'Player' object and lowercase it
+     * at the sametime
+     * @param input (E.g. "luka doncic")
+     * @return "Luka Dončić"
+     */
+    public static String normalize(String input) {
+        String normalized = Normalizer.normalize(input, Normalizer.Form.NFD);
+        Pattern pattern = Pattern.compile("\\p{InCombiningDiacriticalMarks}+");
+        return pattern.matcher(normalized).replaceAll("").toLowerCase();
+    }
+
+    @GetMapping("/player")
+    public Player getPlayerByPick(@RequestParam int numberOfPick) {
+        return players.stream()
+                .filter(player -> player.getNumberOfPick() == numberOfPick)
+                .findFirst()
+                .orElse(null);
+    }
+
+    @GetMapping("/players")
+    public List<Player> getAllPlayers() {
+        return players;
+    }
+
+    @PostMapping("/players")
+    public Player addPlayer(@RequestBody Player newPlayer) {
```

Comment:
```suggestion
    public ResponseEntity<Player> addPlayer(@RequestBody Player newPlayer) {
```

El status code de cuando se crea un recurso es un 201 CREATED y no un 200 OK. Usa ResponseEntity para retornar lo esperado